components/jumbotron/index.tsx

```diff
@@ -5,14 +5,14 @@ import styles from './Jumbotron.module.css'
 const Jumbotron = () => {
   return (
     <ComponentWrapper height="48.375vw" minHeight="40.5vw" maxHeight="86vh" justify="flex-end" margins="0" style={{
-      backgroundImage: "url(/jumbotron-bg.svg)", backgroundPosition: 'bottom', backgroundSize: 'cover',
+      backgroundImage: "url(/jumbotron-bg.png)", backgroundPosition: 'bottom', backgroundSize: 'cover',
```

Comment:
Add dark overlay to background image so white text on top is more visible and legible